minio/minio
# Added graphs for KMS metrics

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license] (https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description


## Motivation and Context


## How to test this PR?
Load the dashboard in grafana and should see below graphs in MinIO dashboard
![Screenshot from 2023-10-26 13-24-37](https://github.com/minio/minio/assets/6771252/cfdd0460-1f97-4704-9215-c3d855696485)

This needs KES to be setup and enabled for MinIO deployment. Also make sure MinIO has access to KES metrics by enabling below configurations under KES
![image](https://github.com/minio/minio/assets/6771252/0243e9b0-1769-431f-9e06-2f6de6fa0eaf)



## Types of changes

## Checklist:
